aspnetcore/tutorials/publish-to-azure-webapp-using-vs.md

```diff
@@ -16,11 +16,9 @@ uid: tutorials/publish-to-azure-webapp-using-vs
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT), [Cesar Blum Silveira](https://github.com/cesarbs), and [Rachel Appel](https://twitter.com/rachelappel)
 
-## Set up the development environment
+## Set up
 
-* Install the latest [Azure SDK for Visual Studio](https://www.visualstudio.com/vs/azure-tools/). The SDK installs Visual Studio if you don't already have it.
-
-* Verify your [Azure account](https://portal.azure.com/). You can [open a free Azure account](https://azure.microsoft.com/pricing/free-trial/) or [Activate Visual Studio subscriber benefits](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/).
+* Open a [free Azure account](https://azure.microsoft.com/en-us/free) if you do not have one. 
```

Comment:
Can we get this updated to include the tracking query string parameter? For example see https://docs.microsoft.com/en-us/azure/app-service/app-service-web-get-started-dotnet#prerequisites